replica/database.hh

```diff
@@ -1288,6 +1288,15 @@ public:
         }
     };
 
+    using ks_cf_t = std::pair<sstring, sstring>;
+    using ks_cf_to_uuid_t =
+        flat_hash_map<ks_cf_t, table_id, utils::tuple_hash, string_pair_eq>;
+    class column_families_data {
+        rwlock _cf_lock;
+    public:
+        std::unordered_map<table_id, lw_shared_ptr<column_family>> _column_families;
+        ks_cf_to_uuid_t _ks_cf_to_uuid;
```

Comment:
As puvlic members, these need not begin with an underscore.
Or, they can be declared private and the class can provider accessors to  get a reference to them.

Reply:
I see they are made private at the end of the series.
Worth a FIXME comment in the earlt patch.